portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,37 +20,48 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  ArrayList<String> messages;
+  HashMap<String, String> comments;
   
   @Override
   public void init() {
-    messages = new ArrayList<String>();
-    messages.add("Hello, my name is Andre!");
-    messages.add("I would ask you your name, but I dont have user input yet!");
-    messages.add("This is a hardcoded message.");
-    messages.add("Eventually I will have real comments to show.");
-    messages.add("I'm running out of message ideas so I will stop.");
+    comments = new HashMap<String, String>();
+    comments.put("Andre Battle", "Hi again! This is just a hardcoded, test commnent.");
   }
    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertToJson(messages);
+    String json = convertToJson(comments);
     response.setContentType("application/jsom;");
     response.getWriter().println(json);
   }
 
-  private String convertToJson(ArrayList messages) {
+
+  private String convertToJson(HashMap<String, String> comments) {
     String json = "[";
-    for (int i = 0; i < 5; i++) {
-       json += "\"" + messages.get(i) + "\", "; 
+    // String[] keys = comments.keySet().toArray(new String[comments.size()]);
+    for (Map.Entry<String, String> entry: comments.entrySet()) {
+      String name = entry.getKey();
+      String comment = entry.getValue();
+      json += "{\"name\": ";
+      json +=  "\"" + name + "\", "; 
+      json += "\"comment\": ";
+      json +=  "\"" + comment + "\"}, "; 
```

Comment:
With this much formatting, consider using String.format(). It will likely make things easier to read.

https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#format(java.lang.String,%20java.lang.Object...)